install.sh

```diff
@@ -1,16 +1,35 @@
 #!/usr/bin/env bash
 
-set -e
+set -eu -o pipefail
 
-if [[ ! -z $DEBUG ]]; then
+if [[ ! -z ${DEBUG-} ]]; then
     set -x
 fi
 
-if [[ $EUID != 0 ]]; then
+: ${PREFIX:=/usr/local}
```

Comment:
Sorry, but my bash is a little weak. Would you mind explaining what this line is doing?

Reply:
Sure:
- the `:` makes the rest of the line evaluate but discards the output (we don't need the variable value there)
- the `${VAR:=value}` construct is a shell variable expansion which will check if `$VAR` is set (and not empty), and if it isn't it will assign it the `value` given